qutebrowser/browser/hints.py

```diff
@@ -959,3 +975,112 @@ def on_mode_left(self, mode):
             # hinting.
             return
         self._cleanup()
+
+
+class WordHinter:
+
+    """Generator for word hints.
+
+    Class attributes:
+
+    Attributes:
+
+    """
+
+    FIRST_ALPHABETIC = re.compile('[A-Za-z]{3,}')
```

Comment:
Why not move this to `tag_words_to_hints` as well? `re` has a cache, so I don't think it'll make any performance difference, and it's not used anywhere outside.